Rakefile

```diff
@@ -37,9 +37,3 @@ desc "Run tests in docker"
 task test_in_docker: :build_dev do
   sh "docker run -it --rm zendesk/samson_secret_puller-dev rake"
 end
-
-desc "Release a new version"
```

Comment:
why are we getting rid of this?

Reply:
should not be used anymore since we have auto-build, see Readme